src/app/games/gipf/GipfMinimax.ts

```diff
@@ -2,14 +2,15 @@ import { Player } from 'src/app/jscaip/Player';
 import { ArrayUtils, Table } from 'src/app/utils/ArrayUtils';
 import { GipfCapture, GipfMove, GipfPlacement } from './GipfMove';
 import { GipfState } from './GipfState';
-import { PlayerMetricsMinimax } from 'src/app/jscaip/Minimax';
+import { Minimax, PlayerMetricHeuristic } from 'src/app/jscaip/Minimax';
 import { GipfRules, GipfNode, GipfLegalityInformation } from './GipfRules';
+import { MoveGenerator } from 'src/app/jscaip/MGPNode';
 
-export class GipfMinimax extends PlayerMetricsMinimax<GipfMove, GipfState, GipfLegalityInformation> {
-    public static getPossibleCaptureCombinationsFromPossibleCaptures(
-        possibleCaptures: GipfCapture[],
-    ): Table<GipfCapture> {
-        const intersections: number[][] = GipfMinimax.computeIntersections(possibleCaptures);
+export class GipfMoveGenerator extends MoveGenerator<GipfMove, GipfState> {
+    public static getPossibleCaptureCombinationsFromPossibleCaptures(possibleCaptures: GipfCapture[])
+    : Table<GipfCapture>
+    {
+        const intersections: number[][] = this.computeIntersections(possibleCaptures);
```

Comment:
enothis for static method please :D

Reply:
awi, c'est un static que j'ai déstatiqué pour me rendre compte qu'il devait être statique, bien vu !